static/src/javascripts/projects/common/modules/experiments/tests/frontend-dotcom-rendering-epic.js

```diff
@@ -0,0 +1,205 @@
+// @flow
+
+import { getBodyEnd } from '@guardian/slot-machine-client';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+import {
+    makeEpicABTest,
+    setupOnView,
+    emitBeginEvent,
+    setupClickHandling,
+    emitInsertEvent,
+} from 'common/modules/commercial/contributions-utilities';
+import reportError from 'lib/report-error';
+import fastdom from 'lib/fastdom-promise';
+import config from 'lib/config';
+import { getMvtValue } from 'common/modules/analytics/mvt-cookie';
+
+const campaignId = 'frontend_dotcom_rendering_epic';
+const geolocation = geolocationGetSync();
+
+const buildKeywordTags = page => {
+    const keywordIds = page.keywordIds.split(',');
+    const keywords = page.keywords.split(',');
+    return keywordIds.map((id, idx) => ({
+        id,
+        type: 'Keyword',
+        title: keywords[idx],
+    }));
+};
+
+const checkResponseOk = response => {
+    if (response.ok) {
+        return response;
+    }
+
+    throw new Error(
+        `Contributions fetch failed with response code: ${response.status}`
+    );
+};
+
+const decodeJson = response => response.json();
+
+const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
+
+const canRun = (): boolean =>
+    config.get('page').dcrCouldRender &&
+    config.get('tests').dotcomRenderingControl === 'control';
+
+const frontendDotcomRenderingTest = {
+    id: 'FrontendDotcomRenderingEpic',
+    campaignId,
+
+    highPriority: true,
+
+    start: '2020-03-13',
+    expiry: '2020-05-13',
+
+    author: 'Andre Silva',
+    description:
+        'A/B test Default Epic on Frontend vs DCR, both from a remote source, to compare Epic performance.',
+    successMeasure: 'Conversion rate',
+    idealOutcome: 'No difference between control and variant',
+
+    audienceCriteria: 'All',
+    audience: 1, // TBC
```

Comment:
This definitely feels a bit scary! We should check with Tom F that our logic here makes sense. This is letting in 100% of users, but the `canRun` should filter out based on the server side test, meaning they will move on to the next configured test.

Reply:
I don't know how `dotcomRenderingControl` gets set, but the logic here is sound. If the `canRun` returns false then we'll try the other a/b tests as normal.